integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/exception/WebClientWithMessageException.java

```diff
@@ -0,0 +1,16 @@
+package com.linkedin.openhouse.javaclient.exception;
+
+import org.springframework.web.reactive.function.client.WebClientException;
+
+/**
+ * An exception thrown in Openhouse clients to indicate an error in a tables API response or
+ * request, acting as a wrapper around WebClientException.
+ */
+public abstract class WebClientWithMessageException extends RuntimeException {
+  private final WebClientException exception;
```

Comment:
There is no need to hold instance of `WebClientException` here since all downstream exception are instances of `WebClient(Request/Response)Exception` and `WebClientException` is a field in child classes. 
The only purpose of this class is to have an abstract parent with function `getMessage` so that childExceptions have to implement it.
You can remove the constructor as well.